vyper/utils.py

```diff
@@ -146,6 +146,16 @@ def in_bounds(cls, type_str, value):
                   'wei', 'finney', 'szabo', 'shannon', 'lovelace', 'ada',
                   'babbage', 'gwei', 'kwei', 'mwei', 'twei', 'pwei', 'contract', 'units']
 
+built_in_functions = ['floor', 'ceil', 'as_unitless_number',
+                      'convert', 'slice', 'len', 'concat',
+                      'sha3', 'method_id', 'keccak256', 'ecrecover',
+                      'ecadd', 'ecmul', 'extract32', 'as_wei_value',
+                      'raw_call', 'RLPList', 'blockhash', 'bitwise_and',
+                      'bitwise_or', 'bitwise_xor', 'bitwise_not',
+                      'uint256_addmod', 'uint256_mulmod', 'shift',
+                      'create_with_code_of', 'min', 'max', 'send',
+                      'selfdestruct', 'raw_log']
```

Comment:
We should see if there is not a more dynamic way of pulling these from `functions.py` ?
I believe there is a dict there that could pull this from. Allowing us to not have to worry about changing it in more than one place later.